### PR TITLE
Added info for Microsoft 365 trials

### PR DESCRIPTION
**Pull request by egallagh**
Hope my formatting is OK

### CODE REVIEW

**Comment by illfated** [comment]
Looks good to me, although I sort of expected a colon (**`:`**) at the end of each header sentence, indicating that the following text follows the given condition.
(I could easily be wrong, though, as I might be thinking of ordinary literature form instead of documentation prose.)

**Comment by egallagh** [comment]
I agree, but I was following the existing format that already exists at the top of the page where it says "Applies to" with no colon.

**Comment by Justinha** [comment]
@greg-lindsay OK to merge?

**Comment by mypil** [comment]
@greg-lindsay - can you please assist to check, approve and merge the content for this PR? 

Thank you.

File: windows/deployment/deploy-m365.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ For Windows 10 deployment, Microsoft 365 includes a fantastic deployment advisor
 
 ## Free trial account
 
+**If you already have a Microsoft services subscription account and access to the Microsoft 365 Admin Center**
+
+From the [Microsoft 365 Admin Center](https://portal.office.com), go to Billing and then Purchase services.
+In the Enterprise Suites section of the service offerings, you will find Microsoft 365 E3 and Microsoft 365 E5 tiles.
+There are "Start Free Trial" options available for your selection by hovering your mouse over the tiles.
```
Comment (nenonix) [inline windows/deployment/deploy-m365.md]: "There are "Start Free Trial" options available for your selection by hovering your mouse over the tiles." 
suggest 
Hover your mouse over the tiles and select one of the "Start Free Trial" options.

**Pull request merged**